release/bin/start.sh

```diff
@@ -15,7 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export COMPONENTS="tablet nameserver apiserver"
+set -e
+
+if [[ $OSTYPE == 'darwin'* ]]; then
```

Comment:
so we should replace all shebang in script with `/bin/bash`, if there is no requirements that a env without bash installed.

Reply:
I have tested /bin/sh supports `OSTYPE` as well.